python/scope.py

```diff
@@ -1854,6 +1880,24 @@ def type_check_command(self, tree):
                         )
                     )
             return command
+        elif command.data == "continue":
+            if self.get_parent_types(["while", "for"]) == None:
+                self.errors.append(
+                    TypeCheckError(
+                        command,
+                        "The command continue can only be used inside while or for loops"
+                    )
+                )
+            return command
+        elif command.data == "break":
+            if self.get_parent_types(["while", "for", "if"]) == None:
+                self.errors.append(
+                    TypeCheckError(
+                        command,
+                        "The command continue can only be used inside if statements or while or for loops"
+                    )
+                )
+            return command
```

Comment:
Hmm, is there a reason why these are in this PR?

Reply:
i messed up things